From 9e2aee3e0e884499293d15b08098ac5abf013a1f Mon Sep 17 00:00:00 2001
From: lijuan <juan.li@spacemit.com>
Date: Mon, 24 Jun 2024 10:08:34 +0800
Subject: [PATCH 0764/1204] deconfig: enable CONFIG_DEBUG to more debug log

log:
[   55.059559] ------------[ cut here ]------------
[   55.059585] WARNING: CPU: 1 PID: 330 at mm/page_alloc.c:4403 __alloc_pages+0x3d4/0x21e2
[   55.059640] Modules linked in:
[   55.059661] CPU: 1 PID: 330 Comm: modprobe Tainted: G        W          6.6.31 #15
[   55.059677] Hardware name: spacemit k1-x deb1 board (DT)
[   55.059686] epc : __alloc_pages+0x3d4/0x21e2
[   55.059700]  ra : __kmalloc_large_node+0x92/0x17a
[   55.059723] epc : ffffffff80545158 ra : ffffffff804bde90 sp : ffffffc802ad3330
[   55.059733]  gp : ffffffff864939e0 tp : ffffffd910f23000 t0 : 0000000000000000
[   55.059741]  t1 : ffffffc802ad3440 t2 : 0000000000000001 s0 : ffffffc802ad35b0
[   55.059749]  s1 : 0000000000000000 a0 : 0000000000040cc0 a1 : 000000000000000b
[   55.059757]  a2 : fffffffff2f2f2f2 a3 : 0000000000000000 a4 : 0000000000000001
[   55.059766]  a5 : ffffffff863fe3a5 a6 : fffffffff2000000 a7 : fffffffff1f1f1f1
[   55.059774]  s2 : 0000000000800000 s3 : 0000000000040cc0 s4 : 000000000000000b
[   55.059780]  s5 : ffffffc80a37f910 s6 : ffffffc80932a7a8 s7 : 0000000000100000
[   55.059788]  s8 : ffffffc809370538 s9 : ffffffc802ad3540 s10: dfffffff00000000
[   55.059796]  s11: 000000000000000b t3 : ffffffff80544d84 t4 : 1ffffff90055a6e4
[   55.059803]  t5 : fffffff80055a6d1 t6 : ffffffd90bacc518
[   55.059810] status: 0000000200000120 badaddr: 0000000000000000 cause: 0000000000000003
[   55.059820] [<ffffffff80545158>] __alloc_pages+0x3d4/0x21e2
[   55.059834] [<ffffffff804bde90>] __kmalloc_large_node+0x92/0x17a
[   55.059850] [<ffffffff804be4aa>] __kmalloc+0x6e/0x1ea
[   55.059864] [<ffffffff800135e8>] apply_relocate_add+0x1a0/0x122e
[   55.059886] [<ffffffff801a44b6>] load_module+0x304a/0x5318
[   55.059902] [<ffffffff801a6a16>] init_module_from_file+0xda/0x12e
[   55.059913] [<ffffffff801a6daa>] __riscv_sys_finit_module+0x28a/0x686
[   55.059924] [<ffffffff8000b72e>] syscall_handler+0x64/0xf0
[   55.059937] [<ffffffff8346735c>] do_trap_ecall_u+0x9c/0x9e
[   55.059960] [<ffffffff83483660>] ret_from_exception+0x0/0x64
[   55.059979] ---[ end trace 0000000000000000 ]---

Change-Id: I240f004b5854ed05bf2493b9e2b6a41d810a5be9
---
 arch/riscv/configs/k1_defconfig | 1 -
 1 file changed, 1 deletion(-)

diff --git a/arch/riscv/configs/k1_defconfig b/arch/riscv/configs/k1_defconfig
index 4db3a825313a..55442e7ef71d 100644
--- a/arch/riscv/configs/k1_defconfig
+++ b/arch/riscv/configs/k1_defconfig
@@ -37,7 +37,6 @@ CONFIG_INITRAMFS_SOURCE="rootfs.cpio.gz"
 # CONFIG_RD_LZ4 is not set
 # CONFIG_RD_ZSTD is not set
 CONFIG_EXPERT=y
-# CONFIG_BUG is not set
 CONFIG_KALLSYMS_ALL=y
 CONFIG_PERF_EVENTS=y
 CONFIG_SOC_SPACEMIT=y
-- 
2.47.0

